app/homepage/index.html

```diff
@@ -5,8 +5,23 @@
         <meta name="description" content="">
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <link rel="stylesheet" href="./styles.css">
+        <noscript>
+            <style>
+                #navbar {
+                    top: 120px;
+                }
+
+                h1 {
+                    margin-top: 150px;
+                }
```

Comment:
Ohh,  interesting idea. As ever, I'd be tempted to keep this CSS in another file

Though tbh given how little there is, I'd be more tempted just to lump it in the main CSS file.